data/indicator_coefficient_importer/Makefile

```diff
@@ -1,5 +1,8 @@
 all:
-	make blue-water-indicator-coefficients
+	make blue-water-indicator-coefficients grey-water-indicator-coefficients
 
 blue-water-indicator-coefficients:
-	python indicator_coefficient_importer.py data/bwfp_indicator_coefficients.csv UWU_T
\ No newline at end of file
+	python indicator_coefficient_importer.py data/bwfp_indicator_coefficients.csv UWU_T
+
+grey-water-indicator-coefficients:
+	python indicator_coefficient_importer.py data/bwfp_indicator_coefficients.csv WQ
```

Comment:
shouldn't it be `gwfp_indicator_coefficients.csv` ?

Reply:
yes, sorry. I'm going to change it. and WQ is the short name of the new water quality indicator. We should have that reference in the indicator csv.